resources/js/mp-history/score.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import FlagCountry from 'components/flag-country';
+import Mod from 'components/mod';
+import Ruleset from 'interfaces/ruleset';
+import ScoreJson, { ScoreStatisticsAttribute } from 'interfaces/score-json';
+import UserJson from 'interfaces/user-json';
+import { route } from 'laroute';
+import * as React from 'react';
+import { classWithModifiers } from 'utils/css';
+import { formatNumber } from 'utils/html';
+import { trans } from 'utils/lang';
+
+interface Props {
+  mode: Ruleset;
+  score: ScoreJson;
+  users: Partial<Record<number, UserJson>>;
+}
+
+const firstRow = ['combo', 'accuracy', 'score'];
+const secondRow: ScoreStatisticsAttribute[] = ['count_geki', 'count_300', 'count_katu', 'count_100', 'count_50', 'count_miss'];
+
+export default function Score(props: Props) {
+  const user = props.users[props.score.user_id];
+
+  if (user == null) {
+    throw new Error('user for score is missing');
+  }
+
+  if (props.score.match == null) {
+    throw new Error('score is missing match data');
+  }
+
+  return (
+    <div className='mp-history-game__player-score mp-history-player-score'>
+      <div
+        className='mp-history-player-score__shapes'
+        style={{ backgroundImage: `url(/images/layout/mp-history/shapes-team-${props.score.match.team ?? 'none'}.svg)` }} />
+      <div className='mp-history-player-score__main'>
+        <div className={classWithModifiers('mp-history-player-score__info-box', ['user'])}>
+          <div className='mp-history-player-score__username-box'>
+            <a className='mp-history-player-score__username' href={route('users.show', { user: user.id })}>{user.username}</a>
+            {!props.score.match.pass && <span className='mp-history-player-score__failed'>{trans('matches.match.failed')}</span>}
+          </div>
+          <a href={route('rankings', { country: user.country?.code, mode: props.mode, type: 'performance' })}>
+            <FlagCountry country={user.country} modifiers={'medium'} />
+          </a>
+        </div>
+        <div className={classWithModifiers('mp-history-player-score__info-box', ['stats'])}>
+          <div className={classWithModifiers('mp-history-player-score__stat-row', ['first'])}>
+            <div className='mp-history-player-score__mods'>
+              {props.score.mods.map((mod) => <Mod key={mod} mod={{ acronym: mod }} />)}
+            </div>
+            {firstRow.map((m) => {
+              let modifier = 'medium';
+              let value;
+
+              switch (m) {
+                case 'combo':
+                  value = formatNumber(props.score.max_combo);
+                  break;
+
+                case 'accuracy':
+                  value = formatNumber(props.score.accuracy * 100, 2, { style: 'percent' });
```

Comment:
percent style already handles the `* 100` internally 👀

Reply:
🤦 I even opened my test match page and checked on the failed thing but like completely forgot to double check the percent formatting